numpy/core/function_base.py

```diff
@@ -115,17 +115,24 @@ def linspace(start, stop, num=50, endpoint=True, retstep=False, dtype=None):
     y = _nx.arange(0, num, dtype=dt)
 
     delta = stop - start
+    # In-place multiplication y *= delta/div is faster, but prevents the multiplicant
+    # from overriding what class is produced, and thus prevents, e.g. use of Quantities,sSee gh-7142,
```

Comment:
Sorry to ask for yet another iteration, but the copy/pasting seems to have gone wrong here... Just move "See gh-7142" to the next line...

Reply:
@mhvk Not a problem, sorry for the sloppiness. I have forced pushed an amendment.